### PR TITLE
Update ping command timeout to 4 sec

### PR DESCRIPTION
**Pull request by wenchma**
Signed-off-by: Wen Cheng Ma wenchma@cn.ibm.com

### CODE REVIEW

**Comment by wenchma** [comment]
@coolljt0725 @thaJeztah  this is as discussed in https://github.com/docker/docker/pull/20903

File: integration-cli/docker_cli_network_unix_test.go [hunk 1]
```diff
@@ -1417,7 +1417,7 @@ func (s *DockerSuite) TestDockerNetworkInternalMode(c *check.C) {
 	c.Assert(waitRun("first"), check.IsNil)
 	dockerCmd(c, "run", "-d", "--net=internal", "--name=second", "busybox", "top")
 	c.Assert(waitRun("second"), check.IsNil)
-	_, _, err := dockerCmdWithTimeout(time.Second, "exec", "first", "ping", "-c", "1", "www.google.com")
+	_, _, err := dockerCmdWithError("exec", "first", "ping", "-W", "4", "-c", "1", "www.google.com")
 	c.Assert(err, check.NotNil)
```
Comment (thaJeztah) [inline integration-cli/docker_cli_network_unix_test.go]: do we need to check the output of the error itself? (just wondering if checking for "any" error is right), but perhaps it was like this already

Comment (coolljt0725) [inline integration-cli/docker_cli_network_unix_test.go]: @thaJeztah  IMO,  we should do that. Just as we have done in other tests, we always check `c.Assert(err, check.NotNil)` first and then check the `err` message if it has the expected error.

Comment (wenchma) [inline integration-cli/docker_cli_network_unix_test.go]: @thaJeztah @coolljt0725 your comments are good improvement, will update this.

**Comment by calavera** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM, thanks @wenchma

**Pull request merged**